include/tvm/relay/expr.h

```diff
@@ -539,6 +539,25 @@ class TempExprNode : public ExprNode {
 
 RELAY_DEFINE_NODE_REF(TempExpr, TempExprNode, Expr);
 
+class Annotate;
+class AnnotateNode : public ExprNode {
```

Comment:
Previously we directly use special op as annotation, let us keep that way and keep the expr stable